packages/frameworks/frameworks.json

```diff
@@ -309,6 +309,33 @@
       }
     }
   },
+  {
+    "name": "Storybook",
+    "slug": "storybook",
+    "demo": "https://now-storybook-one.now.sh/",
+    "tagline": "Build bulletproof UI components faster.",
+    "description": "A static deployment of Storybook rendering your components.",
+    "website": "https://storybook.js.org/",
+    "detectors": {
+      "every": [
+        {
+          "path": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@storybook\\/.+\":\\s*\".+?\"[^}]*}"
```

Comment:
I think that detecting a `build-storybook` script would be more reliable than the `devDep` right? Our goal is to detect projects that deploy Storybook in isolation, as upcoming monorepo support should handle the side-by-side case.

Reply:
I made the change, but - FWIW - even people that are using Storybook in conjunction with apps will have that script name

Reply:
I don't think we can use this regex because we also need to match the dependency in `@now/static-build`.

Let's change it to use the typical dependency (whichever is providing the cli `build-storybook` in node_modules).

Reply:
Cool. So @storybook/react is probably good